pinot-segment-local/pom.xml

```diff
@@ -156,5 +156,30 @@
       <type>test-jar</type>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.carrotsearch</groupId>
+      <artifactId>hppc</artifactId>
+      <version>0.7.2</version>
+    </dependency>
+    <dependency>
+      <groupId>org.openjdk.jmh</groupId>
```

Comment:
We keep all the micro-benchmark in the `pinot-perf` package. Let's not introducing jmh in the production code package but move the perf class into the `pinot-perf`